package.json

```diff
@@ -58,23 +58,23 @@
     "@fontsource-variable/inter": "5.0.8",
     "@img/sharp-linux-x64": "^0.33.0",
     "@jscutlery/semver": "^4.2.0",
-    "@k11r/nx-cloudflare-wrangler": "2.7.1",
-    "@nx/cypress": "17.1.3",
-    "@nx/devkit": "17.1.3",
-    "@nx/eslint": "17.1.3",
-    "@nx/eslint-plugin": "17.1.3",
-    "@nx/jest": "17.1.3",
-    "@nx/js": "17.1.3",
-    "@nx/playwright": "17.1.3",
-    "@nx/plugin": "17.2.3",
-    "@nx/storybook": "17.1.3",
-    "@nx/vite": "17.1.3",
-    "@nx/workspace": "17.1.3",
+    "@k11r/nx-cloudflare-wrangler": "3.0.0-feat-sst-upgrade.1",
```

Comment:
this is a weird version name. what is sst-upgrade? surely it's not the sst project

Reply:
it's the latest version -> https://github.com/klaascuvelier/nx-additions/tags

Perhaps we should use 2.8.0?